README.md

```diff
@@ -612,31 +612,52 @@ $ git clone https://github.com/encukou/freezeyt
 3. Add your forked repo as a new remote to your local computer:
 
 ```shell
-$ git remote add <name_your_remote> https://github.com/<your_username>/freezeyt
+$ git remote add <remote_label> git@github.com:<username>/freezeyt.git
 ```
 
-4. Create new branch at your local computer
+4. Create a new branch at your local computer
 
 ```shell
-$ git branch <name_new_branch>
+$ git branch <branch_name>
 ```
 
 5. Switch to your new branch
 
 ```shell
-$ git switch <name_new_branch>
+$ git switch <branch_name>
 ```
 
-6. Make some awesome changes in code
-7. Push changes to your forked repo on GitHub
+6. Update the code
+7. Push the changes to your forked repo on GitHub
 
 ```shell
-$ git push <your_remote> <your_new_branch>
+$ git push <remote_label> <branch_name>
 ```
 
-8. Finally make a pull request from your GitHub account to origin
+8. Finally, make a pull request from your GitHub account to origin
 
-9. Repeat this process until we will have done amazing freezer
+
+### Install dependecies
+
+Project consists from four groups of dependecies:
+1. for freezeyt itself - `.`
+2. for blog - `blog`
+3. for development - `dev`
+4. for mypy type checks - `typecheck`
+
+Each group can be installed, separately,
+`$ python -m pip install -e .[blog]`
+
+`$ python -m pip install -e .[typecheck]`
+
+or all groups at once.
+`$ python -m pip install -e .[blog, dev, typecheck]`
+
+**Note**
+When you get an error `no matches found: .[dev]` it means that
+your shell (e.g.`zsh`) takes the brackets as special chars.
+It helps to quote the brackets:
+`$ python -m pip install -e .'[dev]'`
```

Comment:
Double quotes (`"`) will work in Bash, and according to a quick search in  Windows as well. Could you test them with zsh, and if they work, use them in the examples instead of adding a note?

Reply:
https://github.com/encukou/freezeyt/pull/265/commits/1f87617538bd440bd53d0de25cbd0671200d20cf